### PR TITLE
fix signed operations helper

### PR DESCRIPTION
**Pull request by ayrat555**
fixes https://github.com/poanetwork/mana/issues/291

### CODE REVIEW

File: apps/blockchain/test/blockchain/state_test.exs [hunk 1]
```diff
@@ -1927,7 +1927,7 @@ defmodule Blockchain.StateTest do
     # "stSystemOperationsTest/suicideCoinbase.json",
     "stSystemOperationsTest/suicideNotExistingAccount.json",
     "stSystemOperationsTest/suicideOrigin.json",
-    # "stSystemOperationsTest/suicideSendEtherPostDeath.json",
+    "stSystemOperationsTest/suicideSendEtherPostDeath.json",
```
Comment (germsvel) [inline apps/blockchain/test/blockchain/state_test.exs]: That's strange. Shouldn't this have already been uncommented? Did we forget to do that @masonforest?

File: apps/evm/lib/helpers.ex [hunk 2]
```diff
@@ -7,6 +7,9 @@ defmodule EVM.Helpers do
   use Bitwise
   alias EVM.Address
 
+  @max_uint_255 round(:math.pow(2, 255)) - 1
+  @max_256_ceiling round(:math.pow(2, 256))
```
Comment (germsvel) [inline apps/evm/lib/helpers.ex]: Just out of curiosity, does the 256 limit come from the yellow paper?

File: apps/evm/lib/helpers.ex [hunk 3]
```diff
@@ -97,17 +100,15 @@ defmodule EVM.Helpers do
...
+    end
   end
 
   def decode_signed(n) when is_binary(n) do
-    <<sign::size(1), _::bitstring>> = n
-
-    if sign == 0 do
-      :binary.decode_unsigned(n)
-    else
-      :binary.decode_unsigned(n) - EVM.max_int()
```
Comment (germsvel) [inline apps/evm/lib/helpers.ex]: Do we use `EVM.max_int()` anywhere else or can that function now be removed?

Comment (ayrat555) [reply apps/evm/lib/helpers.ex]: yes, it's used

File: apps/evm/lib/helpers.ex (hunk 2 above)
Comment (ayrat555) [reply apps/evm/lib/helpers.ex]: yep, 32 bytes

File: apps/blockchain/test/blockchain/state_test.exs (hunk 1 above)
Comment (ayrat555) [reply apps/blockchain/test/blockchain/state_test.exs]: yes, it looks like so

**Pull request merged**